pages/health-care/my-health-account-validation.md

```diff
@@ -0,0 +1,5 @@
+---
+title: My Health Account Validation
+layout: page-react.html
+entryname: my-health-account-validation
+---
```

Comment:
If you're going to be testing this in lower environments, it might be worth keeping this out of production builds by adding `vagovprod: false`. But up to you 🙂